internal/quickstart/container.go

```diff
@@ -193,23 +193,37 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		)
 		cmd = m.currentModel.Init()
 		m.currentStep += 1
-		sendEvent = true
+		shouldSendTrackingEvent = true
 	default:
 		// ignore other messages
 	}
 
-	if sendEvent {
-		cmd = tea.Batch(cmd, trackSetupStepStartedEvent(m.analyticsTracker, m.currentStep.String()))
```

Comment:
Don't send this every time since toggling the flag will send it each time, not just when user gets to toggle flag page.